composer.json

```diff
@@ -80,14 +80,69 @@
     },
     "prefer-stable": true,
     "scripts": {
-        "analyse": "./vendor/bin/phpstan analyse -c phpstan.neon -l max src",
-        "check-style": "./vendor/bin/ecs check src/ spec/",
-        "fix-style": "./vendor/bin/ecs check src/ spec/ --fix",
+        "all": [
+            "@checks",
+            "@tests"
+        ],
+        "analyse": [
+            "@ensure-test-container-exists",
+            "vendor/bin/phpstan analyse -c phpstan.neon -l max src spec tests/Application/Mapper"
+        ],
+        "assets": [
+            "@ensure-node-modules-installed",
+            "@ensure-assets-compiled",
+            "@ensure-assets-installed"
+        ],
+        "check-style": "./vendor/bin/ecs check src spec tests/Application/Mapper",
+        "checks": [
+            "composer validate --strict",
+            "composer normalize --dry-run",
+            "@check-style",
+            "@analyse"
+        ],
+        "ensure-assets-compiled": "[[ -d tests/Application/public/assets ]] || (cd tests/Application && yarn build)",
+        "ensure-assets-installed": [
+            "[[ -d public/bundles ]] || composer ensure-env-copied && (cd tests/Application && bin/console assets:install --symlink --relative public -e ${SYMFONY_ENV:-'dev'})"
+        ],
+        "ensure-database-created": [
+            "composer ensure-env-copied",
+            "(cd tests/Application && bin/console doctrine:database:create --if-not-exists -e ${SYMFONY_ENV:-'dev'})"
+        ],
+        "ensure-env-copied": "([[ ${SYMFONY_ENV:-'dev'} == 'dev' ]] && composer ensure-env-dev-copied) || ([[ ${SYMFONY_ENV:-'dev'} == 'test' ]] && composer ensure-env-test-copied) || echo 'Unknown environment ${SYMFONY_ENV}'",
+        "ensure-env-dev-copied": "(cd tests/Application && ([[ -f .env.dev.local ]] || cp .env  .env.dev.local))",
+        "ensure-env-test-copied": "(cd tests/Application && ([[ -f .env.test.local ]] || cp .env.test .env.test.local))",
+        "ensure-node-modules-installed": "[[ -d tests/Application/node_modules ]] || (cd tests/Application && yarn install)",
+        "ensure-schema-updated": [
+            "composer ensure-env-copied",
+            "(cd tests/Application && bin/console doctrine:schema:update --force -e ${SYMFONY_ENV:-'dev'})"
+        ],
+        "ensure-test-container-exists": "[[ -f tests/Application/var/cache/test/ApplicationTests_Setono_SyliusMiintoPlugin_Application_KernelTestDebugContainer.xml ]] || (cd tests/Application && php -d memory_limit=-1 bin/console cache:clear --env=test)",
```

Comment:
The test container isn't needed for PHPStan to work anymore.

Reply:
Thanks